cmd/gen-go-sample/primval.go

```diff
@@ -45,7 +45,7 @@ var validPrims = [...]func(string) bool{
 }
 
 func validStrings(s string) bool {
-	return true
+	return s[0] == '"'
```

Comment:
Should we check that the last character is a quote as well? I suspect this is fine as is, because this is an initial sanity check before we rely on the scanner to tokenize fully.

Reply:
Yeah, but that's a good point, it does not hurt to check the last character. I'll add it.